### PR TITLE
fix(netdata): use upstream remover

### LINKED ISSUE #593: Netdata remover can leave netdata processes running behind
## Bug description
After removing netdata, updater processes keep running behind.
The process should follow this guide instead https://learn.netdata.cloud/docs/agent/packaging/installer/uninstall

## Reproduction steps
1. Install netdata
2. remove netdata
3. Find running netdata updater processes

## Installed applications through swizzin
(e.g. Nginx, ffmpeg, panel)

## Expected behavior
Full removal with nothing left behind

## Server Info
**(please complete the following information):**
 - OS: (e.g. Debian 10, Ubuntu 20.04)
 - Arch: 
 - Image Source: (e.g. "Vanilla download" or "Installscript from specific provider")

## Is Swizzin up to date?
(Please run `box update`, and report whether that was succesful. Please try to reproduce your issue) again. mention the commit hash you are seeing)

(e.g. HEAD is now at d4d151b)

## Additional context
(Add any other context about the problem here. Logs, terminal output, etc.)

(Good sources for log output could be the following: (only if they have relevant information))
- `cat /root/logs/install.log`
- `cat /root/logs/swizzin.log`
- `cat /var/logs/nginx/error.log`
- `systemctl status <app>@<(user)>`
- `journalctl -xe`

**(Please paste your code into code blocks using the codeblock format like below)**

(For logs of 50+ lines, please rehost to a paste hoster like 0bin)

 ```
 ... logs ...
 ```

### PR DESCRIPTION
**Pull request by flying-sausages**
## Description
Use actual uninstaller script from netdata instead of removing things one by one

## Fixes issues: 
- Closes #593

## Proposed Changes:
- fix: apply suggestion from #593
- fix: clean up double responsabilities, catch possible failures

## Change Categories

- Bug fix 
- New feature 

## Checklist

- [x] Branch was made off the `develop` branch and the PR is targetting the `develop` branch
- [x] Docs have been made OR are not necessary
    - PR link: 
- [x] Changes to panel have been made OR are not necessary
    - PR link: 
- [x] Code is formatted [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Code conforms to project structure [(See more)](https://swizzin.ltd/dev/structure)
- [x] Shellcheck isn't screaming [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#editor-plugins-and-tooling)
- [x] Prints to terminal are handled [(See more)](https://github.com/swizzin/swizzin/blob/master/CONTRIBUTING.md#printing-into-the-terminal)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Testing was done
   - [x] Tests created or no new tests necessary
   - [x] Tests executed

### Architectures

|   			| `amd64` 	| `armhf` 	| `arm64` 	| Unspecified 	|
|--------		|-------- 	|-------- 	|-------- 	|----------		|
| Focal 		|			|			|		yup	|				|
| Bionic		|			|			|			|				|
| Buster		|			|			|			|				|
| Stretch		|			|			|			|				|
| Raspbian  	|	⚫️		|			|	⚫️		|	⚫️			|

### CODE REVIEW

**Comment by brettpetch** [comment]
@flying-sausages does this just need a test and approve? or...?

File: scripts/remove/netdata.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #! /bin/bash
 # Netdata uninstaller for swizzin
 
-systemctl disable -q netdata > /dev/null 2>&1
-systemctl stop -q netdata
-rm -rf $(which netdata)
-rm -rf /etc/netdata
-rm -rf /var/log/netdata
+"${NETDATA_PREFIX}"/usr/libexec/netdata/netdata-uninstaller.sh --yes --env /etc/netdata/.environment -f >> $log 2>&1 || {
```
Comment (liaralabs) [inline scripts/remove/netdata.sh:4]: Why is `${NETDATA_PREFIX}` here if there is no option to define it?

Comment (flying-sausages) [reply scripts/remove/netdata.sh:4]: It could be actually recorded in the ENV

**Comment by flying-sausages** [comment]
> @flying-sausages does this just need a test and approve? or...?

Yup!

File: scripts/remove/netdata.sh (hunk 1 above)
Comment (flying-sausages) [reply scripts/remove/netdata.sh:4]: Actually it cannot, I'm removing this.

**Review by liaralabs — APPROVED** [review]
LGTM as long as it works 😄

**Pull request merged**